Gopkg.toml

```diff
@@ -31,7 +31,8 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/keycloak-client"
-  version = "1.0-rc3"
+  branch = "credentials"
```

Comment:
You really wanted to merge this file ?

Reply:
Created PR (https://github.com/cloudtrust/keycloak-client/pull/18). Thanks to have notice it.